CMakeLists.txt

```
@@ -113,6 +113,7 @@ if (UNIT_TESTING)
   add_subdirectory(sdk/tests/core)
 
   # IoT
+  add_subdirectory(sdk/tests/iot/adu)
```

Would this make it more discovale instead of using the acronym?
```suggestion
  add_subdirectory(sdk/tests/iot/device_update)
```

---

It's a decent idea but probably something we should look at later. Just because there would be a lot of changes and we want this to be as minimal a PR and process as possible.